src/rogatkin/mobile/web/Main.java

```diff
@@ -78,19 +83,22 @@ public void onCreate(Bundle savedInstanceState) {
 		Button button = (Button) findViewById(R.id.button1);
 		button.setOnClickListener(new OnClickListener() {
 
+			@SuppressLint("StaticFieldLeak")
 			public void onClick(View v) {
 				final EditText surl = (EditText) findViewById(R.id.editText1);
 				String userInput = surl.getText().toString();
 				InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
 				imm.hideSoftInputFromWindow(surl.getWindowToken(), 0);
-				if (userInput.length() <= "http://".length()) {
+				if (userInput.length() <= "https://".length()) {

```

Comment:
Since "https" string is used more than one place, it makes sense to make it constant to improve code consistency.